ci-templates/gitlab/image-builder.yml

```diff
@@ -47,6 +47,9 @@ variables:
   T4C_CLIENT_EASE_REMOTE_DEBUG:
     value: "0"
     description: "Build the t4c/client/ease/remote/debug image?"
+  JUPYTER:
+    value: "1"
+    description: "Build the jupyter-notebook image?"
```

Comment:
If you apply my comments below, you would have to add a `JUPYTER_VERSION` variable here, defaulting to `python-3.11`

Reply:
Added a `JUPYTER_VERSION` variable.